saleor/graphql/page/tests/test_page.py

```diff
@@ -760,6 +769,84 @@ def test_create_page_with_page_reference_attribute(
     assert page_type_page_reference_attribute.values.count() == values_count + 1
 
 
+@freeze_time(datetime(2020, 5, 5, 5, 5, 5, tzinfo=pytz.utc))
+def test_create_page_with_date_attribute(
```

Comment:
IMHO this should be split into two tests. You're testing here `date` attribute but also `datetime` attribute.